app/views/publishers/vacancies/select_a_job_for_copying.html.slim

```diff
@@ -2,10 +2,21 @@
 
 .govuk-grid-row
   .govuk-grid-column-full
-    = form_tag redirect_to_copy_job_organisation_jobs_path do
-      .govuk-form-group
-        = vacancy_selector(vacancies.published.order(:job_title), organisation: current_organisation, label_text: t(".heading"))
```

Comment:
if the vacancy_selector component is used only here, why don't we get rid of it?